spec/features/mechanic_show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Mechanic Show Page" do
+  before :each do
+    @amusement_park = AmusementPark.create!(name: "Fun Plex", admission_cost: 10)
+    
+    @mechanic_1 = Mechanic.create!(name: "Jack", years_experience: 10)
+    @mechanic_2 = Mechanic.create!(name: "Jill", years_experience: 20)
+
+    @ride_1 = Ride.create!(name: "Twister", thrill_rating: 7, open: true, amusement_park: @amusement_park)
+    @ride_2 = Ride.create!(name: "Tea Cups", thrill_rating: 4, open: true, amusement_park: @amusement_park)
+    @ride_3 = Ride.create!(name: "El Diablo", thrill_rating: 9, open: false, amusement_park: @amusement_park)
+    @ride_4 = Ride.create!(name: "Log Ride", thrill_rating: 5, open: false, amusement_park: @amusement_park)
+
+    @mech_ride_1 = MechanicRide.create!(mechanic: @mechanic_1, ride: @ride_1)
+    @mech_ride_1 = MechanicRide.create!(mechanic: @mechanic_1, ride: @ride_2)
+    @mech_ride_2 = MechanicRide.create!(mechanic: @mechanic_2, ride: @ride_3)
+  end
+  
+  # User Story 1
+  it "displays their name, years of experience, and the names of all rides they are working on." do
+    visit "/mechanics/#{@mechanic_1.id}"
+    
+    
+    expect(page).to have_content(@mechanic_1.name)
+    expect(page).to have_content(@mechanic_1.years_experience)
```

Comment:
So this test isn't great because it's testing that the number `10` appears _anywhere_ on the page. So that 10 could be in the footer as part of the address, or literally anywhere else. What would help is if you tested this content with a label: `expect(page).to have_content("Years of Experience: #{@mechanic_1.years_experience)}")` that way rspec is looking for that content to be all in one line. Then what would make this even better, would be using within blocks to specific that this years of experience should be next to Mechanic 1's name.